app/browser/api/ledger.js

```diff
@@ -703,8 +703,9 @@ const runPublishersUpdate = (state) => {
 const updatePublishers = (state, publisherKeys) => {
```

Comment:
I know I'm late to the party, but for future ref: 
Is state getting updated at all in `updatePublishers`?

Reply:
Good spot. It doesn't hurt anything at the moment but we can change this in a future PR.